doc/testing_guide.md

```diff
@@ -78,7 +78,7 @@ yarn start
 ### Local parachain (collator)
 Download the `polka-storage` parachain repository:
 ```bash
```

Comment:
Download the `polkadot-native-storage`...

Reply:
Corrected.